test/fixtures/dotenv/valid.env

```diff
@@ -30,6 +30,7 @@ EQUAL_SIGNS=equals==
 RETAIN_INNER_QUOTES={"foo": "bar"}
 RETAIN_INNER_QUOTES_AS_STRING='{"foo": "bar"}'
 RETAIN_INNER_QUOTES_AS_BACKTICKS=`{"foo": "bar's"}`
+RETAIN_INNER_QUOTES_AS_ESCAPE_SLASH=`{\"foo\": \"bar\"}`
```

Comment:
Maybe add e.g. `C:\\Windows\\system32` as a test case?

Reply:
What's the default behavior of dotenv?

Reply:
There was a problem with the current implementation, so I modified it and added the test you commented on.

Reply:
@pluris Thanks for the quick fix! I think there is still an issue. I would expect the test to read:

```
assert.strictEqual(process.env.PATH_WINDOWS, 'C:\\Windows\\system32');
```

instead of the current

```
assert.strictEqual(process.env.PATH_WINDOWS, 'C:\\\\Windows\\\\system32');
```

That is, escaping a backslash should result in just one backslash... This is also what dotenv does, @anonrig.